TUnit.Core.SourceGenerator/Generators/AotConverterGenerator.cs

```diff
@@ -73,6 +73,13 @@ public void Initialize(IncrementalGeneratorInitializationContext context)
             return null;
         }
 
+        // Skip conversion operators for types that are not accessible from generated code
+        // Private and internal types cannot be accessed from the TUnit.Generated namespace
```

Comment:
The comment about internal types not being accessible seems inconsistent with how the method actually handles them. Just removing this line would probably suffice.
```suggestion
```